go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -398,6 +401,8 @@ func (m *MasterController) setupHybridLRPolicySharedGw(nodeSubnets []*net.IPNet,
 				"name": ovntypes.HybridSubnetPrefix + nodeName,
 			},
 		}
+		// TODO(trozet): evaluate what fields to update. The predicate checks all fields set, so doesn't look like there
```

Comment:
What about this TODO? Making sure you intended to leave it around.

Reply:
fixed when I rebased